src/wp-includes/functions.php

```diff
@@ -7665,7 +7665,7 @@ function wp_post_preview_js() {
 		}
 
 		if ( window.addEventListener ) {
-			window.addEventListener( 'unload', function() { window.name = ''; }, false );
+			window.addEventListener( 'pagehide', function() { window.name = ''; } );
```

Comment:
I was confused why the `PageTransitionEvent` being passed to this event handler has its `persisted` property set to `false`, both for the `pagehide` and the `pageshow` events. In looking at DevTools, I see:

> ![image](https://github.com/WordPress/wordpress-develop/assets/134745/e26f70b1-7665-4ed1-a5a9-9ad8fd211a5e)

So it is explained that the fact that `window.open()` was used to open the preview is another way that bfcache is disabled. 

I think this is all good. I'm seeing that the `window.name` is being set to `wp-preview-...` when the tab is opened when clicking preview. When I navigate the preview window to another page, the `window.name` is set to an empty string. When I then re-preview a post I am opened in a new tab rather than the previous preview tab (which is now no longer a preview). If in that preview tab I go back to the previewed post, and then hit preview again, then it is that tab that gets updated with the latest preview content.

So I think we're good here.